smFISH/metrics/visualisation.py

```diff
@@ -0,0 +1,228 @@
+import matplotlib.pyplot as plt
```

Comment:
I'm not sure I understand -- maybe just make whatever changes you think are good and let me look over the code before committing it?

I'm interested in getting rid of everything in the `metrics` directory eventually, and moving it all to `Inference` or `VariationalInference`. Currently, it seems like every metric is defined two places --- in an `Inference` class (or subclass of it) and then again in the `metrics` directory. Seems unnecessarily complicated.

But I could easily be missing something about the importance of the `metrics` directory -- glad to see what you can do to simplify it.

Reply:
Shouldn't all this metrics be separated in different files ? If so would you prefer to move them all in Inference directory ? Or if not, this would mean the `variational_inference.py` file would get pretty big, with all the methods following one another. 

It is true that very few functions would be used elsewhere - there are the clustering scores that might serve for other algorithms.

Reply:
I'm not sure having long files is a problem.